.github/workflows/deploy.yml

```
@@ -34,32 +34,37 @@ jobs:
+        run: echo "DOCKERFILE_NAME=python${{ matrix.image.python_version }}" >> $GITHUB_ENV
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
+      - name: Get date for tags
+        run: echo "DATE_TAG=$(date -I)" >> "$GITHUB_ENV"
+      - name: Build and push
+        uses: docker/build-push-action@v2
```

actionlint (1.7.11)

[error] the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)